index.html

```diff
@@ -283,6 +283,40 @@ <h4 class="about-me">About me</h4>
         </article>
     </section>
 </section>
+<section class="add-contact-text">
+    <div class="text-2">
+            <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    </div>
+    <form class="add-info" action="https://formspree.io/f/xgebvjyw"
+    method="POST">
+        <div class="full-name">
+            <input class="border-none-1" type="text" name="name" maxlength="30" placeholder="Full name" required>
+            <hr class="container-1">
+        </div>
+        <div class="email-address">
+            <input class="border-none-2" type="email" name="email" placeholder="Email address" required>
+            <hr class="container-2">
+        </div>
+        <div class="message-text">
+            <textarea class="border-none-3" name="message" rows="8" cols="39" maxlength="500" placeholder="Enter text here" required></textarea>
+        </div>
+
+        <button class="form-button" type="submit" button-interaction="hover">         
+            <h3 class="form-button-text">Get in touch</h3>
+        </button>

```

Comment:
Please consider adding the correct hover and click functionality on the button to make it closer to the design. Remember the goal of the project is to make a clone of the design. Using your own sense for styles and others is not part of this project.

## Hover(:hover) and Press(:active) styles:-

![image](https://user-images.githubusercontent.com/94223367/227982423-84b00128-b43b-48d6-983d-5dab41d60404.png)

## Conclusion:- 

When the button is hovered or pressed it needs to be exactly in style as the above diagram illustrates.